paper/src/main/java/vg/civcraft/mc/citadel/activity/ChunkCoord.java

```diff
@@ -1,18 +1,6 @@
 package vg.civcraft.mc.citadel.activity;
 
-import org.bukkit.Chunk;
-
-class ChunkCoord {
-	public final short worldId;
-	public final int x;
-	public final int z;
-
-	public ChunkCoord(short worldId, int x, int z) {
-		this.worldId = worldId;
-		this.x = x;
-		this.z = z;
-	}
-
+record ChunkCoord (short worldId, int x, int z) {
 	@Override
 	public boolean equals(Object obj) {
```

Comment:
There's no need for `.equals()` or `.hashCode()` on records unless you're doing something very custom

Reply:
equals - agree
hashCode - I want to have here my own hashing function